### ISSUE TITLE
Add-ons Manager: change the title when all add-ons are disabled via --disable-addons command line option

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#5785]
Hi,

Related to #3090:

Currently when users open add-ons manager after starting NVDA with no add-ons running, NVDA displays, "Add-ons Manager". I propose changing the title to something like "Add-ons Manager (no add-ons running)" to let the user know this is such a case? I'd prefer if i3090 branch can be used for this work also.

Thanks.

**Comment by leonardder** [comment]
This will also require changing the title bar when all add-ons are manually disabled in the gui, so the title bar should be changed based on the actual number of running add-ons and not on whether NVDA is started with all add-ons disabled or not.

As an alternative, how about changing the title bar to %d add-ons running?

**Comment by josephsl** [comment]
Hi, I hope wxPython allows changing the title of a dialog once defined (which is a necessary requirement)… Thanks.

**Comment by jcsteh** [comment]
I disagree that this should occur when all add-ons are manually 
disabled. These are two very different states. When a user starts with 
add-ons disabled, this disables *all* add-ons and is a temporary state. 
In contrast, when the user manually disables an add-on, it is a very 
explicit action for that add-on. Whether it's 1 add-on, 10 add-ons or 
all add-ons makes no difference. To put it another way, there's nothing 
special about all add-ons when the user disables each one explicitly, no 
more than there is if they disable all add-ons except one.

**Comment by leonardder** [comment]
[**@josephsl**](https://github.com/josephsl) commented on [2 Mar 2016, 10:00 CET](https://github.com/nvaccess/nvda/issues/5785#issuecomment-191139826 "2016-03-02T09:00:43Z - Replied by Github Reply Comments"):
> Hi, I hope wxPython allows changing the title of a dialog once defined (which is a necessary requirement)… Thanks.

This is certainly possible, see #7302. I'm marking this as good for new dev

**Comment by josephsl** [comment]
Hi, yep, #577 is a good proof of this. Thanks.

**Comment by lukaszgo1** [comment]
With merge of #9474 this issue is not fixed. Can someone close it?
